phase1/vsphere/Kconfig

```diff
@@ -31,36 +31,36 @@ config phase1.vSphere.datacenter
 	string "Datacenter"
 	default "datacenter"
 	help
-		The datacenter to be used by vSphere cloud provider plugin.
+		The datacenter to be used by vSphere cloud provider plugin. Use vCenter UI or ```govc datacenter.info```
```

Comment:
Can we add a picture with vCenter Navigator window and use same sample values from picture in documentation?